### ISSUE TITLE
How could I deal with this issue when I`m running main.py?

### ISSUE THREAD

**Issue by EvayJacksonChen** [JedMills/MTFL-For-Personalised-DNNs#5]
![image](https://user-images.githubusercontent.com/84269313/179562681-bc4b62bf-c463-4f4c-84a1-6019230e23e9.png)

**Comment by JedMills** [comment]
Hi CoolYesWow,

I've updated the README with more detailed instructions on running the
simulations.

Warm regards,
Jed

**Comment by EvayJacksonChen** [comment]
Hi Jed,

It`s great to hear from you. And this really helps me a lot when I`m stuck in training models! And now I`m starting it~~ It`s pretty cool.  So when I`m done, how could I evaluate the performance of these models while I do not have any Raspberry Pi?

Warmly,
Cool

**Comment by JedMills** [comment]
Hi Cool,

I'm glad it helps! I've also just added the file "Fig-5-settings.md" containing all the hyperparameters used for the experiments in Fig 5, which another user had requested. 

The experiments from Fig. 4, Fig. 5, Table 2 and Table 3 are all simulated (run on a workstation), so if you want to reproduce those you will not need an RPi. Only the results from Table 4 (where we recorded the time taken to run each round of the algorithms) were performed on RPi's. 

Warm regards,
Jed
